app/src/main/java/com/example/euriskocodechallenge/ui/home/news/NewsRVAdapter.kt

```diff
@@ -48,10 +50,13 @@ class NewsRVAdapter : RecyclerView.Adapter<NewsRVAdapter.NewsViewHolder>() {
         holder.binding.apply {
             titleTv.text = newsList[position].title
             authorTv.text = newsList[position].byline
-            newsThumb.load(newsList[position].media[0].mediaMetadata[0].url) {
-                crossfade(true)
-                crossfade(1000)
-                transformations(RoundedCornersTransformation(10F))
+            if(newsList[position].media[0].mediaMetadata[0].url.isNullOrEmpty()){
```

Comment:
@martin371998 already commented in another PR

Reply:
done in another branch